src/sentry/static/sentry/app/views/stream/searchBar.jsx

```diff
@@ -74,11 +75,11 @@ class SearchBar extends React.Component {
   render() {
     return (
       <SmartSearchBar
-        {...this.props}
         onGetTagValues={this.getTagValues}
         supportedTags={TagStore.getAllTags()}
         defaultSearchItems={SEARCH_ITEMS}
         maxSearchItems={5}
+        {...this.props}
```

Comment:
If you're spreading the props down, you could probably remove `supportedTags` on line 79 as it will be reset here on the spread.

Reply:
Good point, I thought this was used in a few other places but turns out that's another search bar component